### PR TITLE
feat: Landing page update (Revised version)

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This is the better-planned revised version of #119. 

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: components/sections/home-page/Hero.tsx [hunk 1]
```diff
@@ -1,47 +1,71 @@
...
+                  {item.ctaLabel}
+                </Button>
+              ))}
+              </div>
+              
+              <div className="pt-6 pb-10">
+                <Typography alignLarge="left" alignSmall="left" variant="body3">
+                  Working with team?
+                  <span>
+                    <Link href="/enterprise">
```
Comment (bdougie) [inline components/sections/home-page/Hero.tsx:51]:
```suggestion
                    <Link href="/enterprise">
```

This link is broken. And can this be `/teams` instead?

**Review by bdougie — CHANGES_REQUESTED** [review]

File: components/sections/home-page/Hero.tsx (hunk 1 above)
Comment (shamimbinnur) [reply components/sections/home-page/Hero.tsx:51]: We do not have a page on '/teams' also.

**Comment by chhristopher** [comment]
Hey Shamim, thanks for this. I think we can go live with this and ship it but I'd like to roll these changes out one by one soon after. 

Changes Page 1/4:
1. Padding
2. H1 in our original takes up only 2 lines versus the 3 lines in the preview
3. Padding/sizing
4. Padding/sizing

Seems like most of the changes here are due to sizing issues (everything is enlarged). None are red flags so we can go live with this, but as stated earlier, would like to see these changes rolled out.

Comparison-1

**Comment by chhristopher** [comment]
Changes Page 2/4:

1. Original design takes up 3 lines versus 4
2. Padding and sizing




Comparison-2

**Comment by chhristopher** [comment]
Changes Page 4/4:

1. Original 2 lines versus preview 3
2. Design element padding. The preview element is scaled up

**That's it. Seems like most of these are scaling differences. None of them are dealbreakers, hence why I think we can go live now, but want to eventually push these changes out when we can. Thanks again @shamimbinnur, great job. 


Comparison-4

File: components/sections/home-page/Hero.tsx [hunk 2]
```diff
@@ -1,47 +1,71 @@
...
+              <div className="pt-6 pb-10">
+                <Typography alignLarge="left" alignSmall="left" variant="body3">
+                  Working with team?
+                  <span>
+                    <Link href="/teams">
+                      <a>
+                        <DecoratedText content="$orange-to-yellow Click here$orange-to-yellow"/>
+                      </a>
+                    </Link>
+                  </span>
```
Comment (bdougie) [inline components/sections/home-page/Hero.tsx:56]:
```suggestion
{/*  needs to be implemented
                  <span>
                    <Link href="/teams">
                      <a>
                        <DecoratedText content="$orange-to-yellow Click here$orange-to-yellow"/>
                      </a>
                    </Link>
                  </span> 
                  */} 
```

File: components/sections/home-page/Hero.tsx [hunk 3]
```diff
@@ -1,47 +1,73 @@
...
+                    <Link href="/teams">
+                      <a>
+                        <DecoratedText content="$orange-to-yellow Click here$orange-to-yellow"/>
+                      </a>
+                    </Link>
+                  </span> 
+                  */} 
+                </Typography>
+              </div>
+            </div>
```
Comment (bdougie) [inline components/sections/home-page/Hero.tsx:61]:
```suggestion
{/*  needs to be implemented

              <div className="pt-6 pb-10">
                <Typography alignLarge="left" alignSmall="left" variant="body3">
                  Working with team?
                  <span>
                    <Link href="/teams">
                      <a>
                        <DecoratedText content="$orange-to-yellow Click here$orange-to-yellow"/>
                      </a>
                    </Link>
                  </span> 
                </Typography>
              </div>
            </div>
                              */} 

```

File: components/sections/home-page/Hero.tsx [hunk 4]
```diff
@@ -1,47 +1,75 @@
...
+                    <Link href="/teams">
+                      <a>
+                        <DecoratedText content="$orange-to-yellow Click here$orange-to-yellow"/>
+                      </a>
+                    </Link>
+                  </span> 
+                </Typography>
+              </div>
+            </div>
+                              */}
```
Comment (bdougie) [inline components/sections/home-page/Hero.tsx:62]:
```suggestion
                </Typography>
              </div>
                                            */} 

            </div>
```

**Comment by bdougie** [comment]
We needed this done on the 1st of May, so I am merging this in. 

@chhristopher and @shamimbinnur I create this issue to re implement the old landing on the `/teams` route. We need that ASAP. 

https://github.com/open-sauced/landing-page/issues/121

**Pull request merged**